app/controllers/api/current/help_controller.rb

```diff
@@ -7,6 +7,21 @@ class Api::Current::HelpController < APIController
   skip_before_action :validate_facility, only: [:show]
   skip_before_action :validate_current_facility_belongs_to_users_facility_group, only: [:show]
 
+  HTML_TRANSLATIONS_DIR = 'config/locales/api/help/*.html'
```

Comment:
Could this be HELP_TRANSLATIONS_DIR instead? 
Since its limited to the help htmls